otelcollector/deploy/chart/prometheus-collector/templates/prometheus-collector-settings-configmap.yaml

```diff
@@ -37,6 +37,17 @@ data:
     windowskubeproxy = {{ .Values.keepListRegexes.windowsKubeProxy | toString | quote }}
   debug-mode: |-
     enabled = {{ .Values.debugMode.enabled }}
+  default-targets-scrape-interval-settings: |-
+    kubelet = {{ .Values.scrapeIntervalTargets.kubelet | toString | quote}}
```

Comment:
can you also log a workitem to add telemetry if any of the default target's scrape interval gets overridden thru config map ? You dont need to hold this PR for that.

Reply:
Added this https://msazure.visualstudio.com/InfrastructureInsights/_workitems/edit/16099871.